scripts/Interstitial_Pages_SFR-1410.py

```diff
@@ -54,9 +54,33 @@ def update_batch(museObject, bucketName, currKey):
 
         # Updating old url link with new link
         museDictObject['readingOrder'][r]['href'] = urlunparse(url_parts)
+        logging.info(museDictObject['readingOrder'])
 
         r += 1
 
+
+    j = 1
+    while j < len(museDictObject['toc']):
+
+        urlLink = museDictObject['toc'][r]['href']
+        logging.info(urlLink)
+
+        # Convert ParseResponse to List object to modify data
+        url_parts = list(urlparse(urlLink))
+        query = dict(parse_qsl(url_parts[4]))
+        logging.info(query)
+
+        # Updating query parameter of url
+        params = {'start': 2}
+        query.update(params)
+        url_parts[4] = urlencode(query)
+
+        # Updating old url link with new link
+        museDictObject['toc'][r]['href'] = urlunparse(url_parts)
```

Comment:
A general practice comment: this could all be refactored into a separate method that contains this logic for `toc` and `readingOrder` because it should be the same. This would make it a bit more maintainable and give us confidence that the same thing is happening in both arrays.

Obviously this works! This is just with an eye towards best practice.

Reply:
Thanks for the advice. I'll keep that in mind for future scripts I write.